tests/docs/running-perf-tests.md

```diff
@@ -140,6 +141,21 @@ make setup-test-env-k6
 make test-perf-all
 ```
 
+You can also run the test apps individually.
```

Comment:
The best way is to actually use DAPR_PERF TEST that allows passing more than one test:

For example, this runs 2 actors test:

```sh
DAPR_PERF_TEST="actor_id_scale actor_type_scale" make test-perf-all
```

Reply:
updated as suggested.